libs/components/forms/src/lib/modules/radio/fixtures/radio.component.fixture.html

```diff
@@ -9,6 +9,8 @@ <h1 id="radio-group-label">Radio label</h1>
       [label]="label1"
       [value]="value1"
       [(ngModel)]="selectedValue"
+      (checkedChange)="onCheckedChange($event)"
+      (disabledChange)="onDisabledChange($event)"
```

Comment:
☝️ Added these for coverage.

Reply:
How does this help coverage? Should we be testing something we're not? If they are needed, you don't need to pass `$event` since the functions you're binding to don't do anything.